adapters/adview/adview.go

```diff
@@ -143,17 +163,17 @@ func (a *adapter) buildEndpointURL(params *openrtb_ext.ExtImpAdView) (string, er
 	return macros.ResolveMacros(a.endpoint, endpointParams)
 }
 
-func getMediaTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType, error) {
-	mediaType := openrtb_ext.BidTypeBanner
-	for _, imp := range imps {
-		if imp.ID == impID {
-			if imp.Video != nil {
-				mediaType = openrtb_ext.BidTypeVideo
-			} else if imp.Native != nil {
-				mediaType = openrtb_ext.BidTypeNative
-			}
-			return mediaType, nil
-		}
+func getMediaTypeForBid(bid openrtb2.Bid) (openrtb_ext.BidType, error) {
+	switch bid.MType {
+	case openrtb2.MarkupBanner:
+		return openrtb_ext.BidTypeBanner, nil
+	case openrtb2.MarkupVideo:
+		return openrtb_ext.BidTypeVideo, nil
+	case openrtb2.MarkupAudio:
```

Comment:
as per `adview` bidder-info yml, audio media type is not supported. should remove audio case

https://github.com/prebid/prebid-server/blob/cd9855403a50d108475dcf66d6d6e734db163be3/static/bidder-info/adview.yaml#L9-L12

Reply:
Thanks for the reminder, fixed.